dependencies.gradle

```diff
@@ -87,9 +87,9 @@ ext.dep = [
   "tracingJaeger": "com.uber.jaeger:jaeger-core:0.24.0",
   "tracingZipkin": "io.opentracing.brave:brave-opentracing:0.29.0",
   "vitess": "io.vitess:vitess-jdbc:3.0.0",
-  "wireRuntime": "com.squareup.wire:wire-runtime:3.0.0-rc01",
-  "wireGrpcClient": "com.squareup.wire:wire-grpc-client:3.0.0-rc01",
-  "wirePlugin": "com.squareup.wire:wire-gradle-plugin:3.0.0-rc01",
+  "wireRuntime": "com.squareup.wire:wire-runtime-multiplatform:3.0.0-rc02",
```

Comment:
why does this change? I don’t like this because it means we could have both!

Reply:
I was just playing around, I think I'll revert this whole file (except for the wire's version upgrade) and will keep everything the same otherwise